src/hugr/validate.rs

```diff
@@ -40,18 +40,30 @@ struct ValidationContext<'a> {
 impl Hugr {
     /// Check the validity of the HUGR.
     pub fn validate(&self) -> Result<(), ValidationError> {
-        let mut validator = ValidationContext::new(self);
+        self.validate_with_extension_closure(HashMap::new())
```

Comment:
please add a todo for validating hugrs with input extension set variables

Reply:
I've added issue #457 after our discussion this morning

Reply:
cool, the comment here can link to that then